src/translations/strings.yaml

```diff
@@ -555,8 +555,8 @@ onboarding:
   progressBarAddDevices:
     value: Devices
     comment: Label for onboarding progress bar step
-  progressBarGetStarted:
-    value: Start
+  progressBarGetStarted2:
+    value: Startup
```

Comment:
The new label feels quite confusing: any explanation from content about the reason for this change?

Reply:
@flodolo Agreed. I think this was copied over from the desktop onboarding, which has as the last step "Startup" and prompts users to turn on "Connect VPN on startup". 

This is a totally different step, though. It seems like a better label here might be "Connect". Thoughts? (If @flodolo is ok with this change, @MattLichtenstein is there an issue with making this update?)

Reply:
`Connect` works for me, there's no space for interpretation

Reply:
No strong opinions here, but for a bit of background: 

The step was originally labelled "Get started" to denote it was the last step. This eventually got shortened to "Start" and it seems like that meaning may have gotten lost in translation when we switched the final slide on desktops content to being about connecting on startup (instead of quick access), leading to the final step being called "Startup." This now has a different meaning than the intended "Get started" and I agree it doesn't make much sense on mobile. I prefer "Connect" but also don't mind going back to "Start" if we wanted to be consistent across platforms (which I don't think is absolutely necessary, since the content IS different, just another option)

Reply:
I think, since this really is a different step from the desktop onboarding, it's better to be more specific here than consistent across platforms. (My understanding is that these labels are acting as navigation labels for screen readers, and "Connect" is definitely a more clear and accurate label for this step than "Start").

Reply:
I think this is unclear also on Desktop. Does it mean "Start the VPN" or something else? I would have no clue how to translate this without additional context; without it, we're going to get very confusing translations.